method: intrinsics_based
fx: 784.6866524961796
fy: 784.6809717665531
cx: 319.07595214284834
cy: 255.5166207673894
scale_meter_per_pixel: 0.0012743940784314075
scale_std: 0.0
origin_pixel: [253.39999389648438, 245.3000030517578]
origin_world: [0.0, 0.0]
pixel_points:
  - [253.39999389648438, 245.3000030517578]
  - [286.79998779296875, 159.3000030517578]
  - [286.79998779296875, 159.3000030517578]
  - [182.49057006835938, 194.40023803710938]
  - [182.49057006835938, 194.40023803710938]
  - [182.49057006835938, 194.40023803710938]
world_points:
  - [0.0, 0.0]
  - [0.133, -0.045]
  - [0.133, -0.045]
  - [-0.174, -0.078]
  - [-0.174, -0.078]
  - [-0.174, -0.078]
homography_matrix:
  - [0.0012743940784314075, -2.63219599529339e-11, -0.2325649058141135]
  - [-7.586329901405127e-12, 0.0012744033479582858, -0.24774431663874527]
  - [-1.635180504161799e-10, -1.3894330565343076e-11, 1.0]